mailpoet/lib/Automation/Integrations/MailPoet/Actions/SendEmailAction.php

```diff
@@ -51,6 +59,15 @@ public function getName(): string {
     return __('Send email', 'mailpoet');
   }
 
+  public function getArgsSchema(): ObjectSchema {
+    return Builder::object([
+      'subject' => Builder::string()->default(__('Subject', 'mailpoet')),
+      'preheader' => Builder::string(),
+      'from_name' => Builder::string()->default($this->settings->get('sender.name')),
+      'email' => Builder::string()->default($this->settings->get('sender.address')),
```

Comment:
Is `'email'` too generic of a name for what I hope I'm correct in assuming is the "From" email address?

Also, is there a reason this doesn't include a newsletter ID? Maybe because the email is never present yet when the action is first created?

Reply:
I actually noticed this just yesterday and fixed it here: https://github.com/mailpoet/mailpoet/pull/4280/commits/dfee6fa39c30420aebd4ec0de4bb1112d9a2a564

I really like what type of details you notice 👏👍

Reply:
Awesome, thanks for making the update!